test/prefer-exponentiation-operator.js

```diff
@@ -0,0 +1,51 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-exponentiation-operator';
+
+const ruleTester = avaRuleTester(test, {
+	parserOptions: {
+		ecmaVersion: 2016
+	}
+});
+
+ruleTester.run('prefer-exponentiation-operator', rule, {
+	valid: [
+		'a ** b;',
+		'2 ** 4;',
+	],
+	invalid: [
+		{
+			code: 'const x = Math.pow(2, 4);',
+			errors: [{message: 'Prefer the exponentiation operator over `Math.pow()`.', column: 11, line: 1}],
```

Comment:
Create a method like [this](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2283cda9313deda0c527ce3af8b7607ad512540f/test/new-for-builtins.js#L11-L14) so you don't have to repeat the error for invalid item. I don't think there's any point in specifying column/line except for the test where there are multiple `Math.pow()` in one test.

Reply:
Since there aren't any parameters, I just made it a constant.